gensim/models/word2vec.py

```diff
@@ -300,7 +295,7 @@ def train_cbow_pair(model, word, input_word_indices, l1, alpha, learn_vectors=Tr
             if w != word.index:
                 word_indices.append(w)
         l2b = model.syn1neg[word_indices]  # 2d matrix, k+1 x layer1_size
-        fb = sigmoid(dot(l1, l2b.T))  # propagate hidden -> output
+        fb = expit(dot(l1, l2b.T))  # propagate hidden -> output
```

Comment:
Is this the only place where inverse logit is used? Probably worth updating all occurrence, consistently.